app/assets/stylesheets/global/_base.scss

```diff
@@ -56,8 +56,30 @@ a {
       color: $color_font-light;
     }
   }
+  &.link-button-row:nth-child(even) {
+    margin: 0;
+    display: block;
+    width: 100%;
+    background-color: $color_green-medium;
+    color: $color_font-light;
+    padding: 0.5em 1em;
+    text-align: center;
+    font-size: $font_size-large;
+    text-decoration: none;
+  }
+  &.link-button-row:nth-child(odd) {
+    margin: 0;
+    display: block;
+    width: 100%;
+    background-color: $color_green-dark;
+    color: $color_font-light;
+    padding: 0.5em 1em;
+    text-align: center;
+    font-size: $font_size-large;
+    text-decoration: none;
+  }
```

Comment:
It's best to have these duplicate rules share the class instead of re-declaring for even/odd rows.

```scss
  &.link-button-row {
    margin: 0;
    display: block;
    width: 100%;
    background-color: $color_green-dark;
    color: $color_font-light;
    padding: 0.5em 1em;
    text-align: center;
    font-size: $font_size-large;
    text-decoration: none;

    &:nth-child(even) { background-color: $color_green-medium; }
  }
```